Makefile

```diff
@@ -320,6 +320,12 @@ test-rpc:
 test-integration:
```

Comment:
are we using these? If not, I guess we can remove them and rename the other to `test-integration`